README.md

```diff
@@ -43,7 +43,7 @@ Overriding with Java System Property:
 -Dcom.cloudbees.jenkins.plugins.advisor.BundleUpload.recurrencePeriodHours=1
 ```
 
-#### CloudBees Jenkins Advisor Upload Timeout
+#### Jenkins Health Advisor by CloudBees Upload Timeout
```

Comment:
Just "Upload Timeout"

Reply:
Please apply same advice below. We don't need to repeat the plugin name for every setting.

Reply:
yes it makes sense

Reply:
fixed